lib/rentlinx/models/property.rb

```diff
@@ -15,15 +16,26 @@ class Property < Base
 
     attr_accessor(*ATTRIBUTES)
 
+    # Posts the property with all of its attached units
```

Comment:
I thought we wanted to get rid of this type of methods that embed the posting of something else with it. If you post a unit here, do you expect it to post with all its component, (amenities, etc.), or just the units?

If we agree to remove these, we could probably do it now.